spec/Section 7 -- Response.md

```diff
@@ -10,71 +10,71 @@ the case that any _field error_ was raised on a field and was replaced with
 
 ## Response Format
 
-A GraphQL request returns either a _response_ or a _response stream_.
+:: A GraphQL _response_ is either a _response map_ or a _response stream_.
 
-### Response
+### Response Map
 
-:: A GraphQL request returns a _response_ when the GraphQL operation is a query
-or mutation. A _response_ must be a map.
+:: A GraphQL request returns a _response map_ when the GraphQL operation is a
+query or mutation. A _response map_ must be a map.
```

Comment:
A response map can be used in many situations. We should try and make the description more accurate. I don't particularly like this edit, but it does seem more accurate.

```suggestion
:: A _response map_ is a map containing data and/or errors returned from GraphQL
execution.

Note: A _response map_ is typically the result type that you would see for a
query or mutation operation, however you may also see it in response to a
subscription operation should a request error occur.
```